inference/enterprise/workflows/complier/steps_executors/models.py

```diff
@@ -187,6 +187,7 @@ def construct_classification_request(
         image=image,
         confidence=resolve(step.confidence),
         disable_active_learning=resolve(step.disable_active_learning),
+        source="workflow-execution",
```

Comment:
ideally would want to pass `source_info` as the workflow id/name, but don't have that context available here easily?

Reply:
yes, we do not have that